src/redempt/redlib/enchants/CustomEnchant.java

```diff
@@ -335,5 +335,16 @@ public String getLore(int level) {
 		}
 		return getDisplayName() + " " + toRomanNumerals(level);
 	}
-	
+
+	/**
+	 *
```

Comment:
Add a description "Increase the level of this enchantment on an item, ensuring it does not exceed the maximum level"